src/apps/exactly/ethereum/exactly.deposit.token-fetcher.ts

```diff
@@ -0,0 +1,27 @@
+import { Register } from '~app-toolkit/decorators';
+import type { GetTokenPropsParams } from '~position/template/app-token.template.types';
+import { Network } from '~types/network.interface';
+
+import type { ExactlyMarketDefinition } from '../common/exactly.definitions-resolver';
+import { ExactlyTokenFetcher } from '../common/exactly.token-fetcher';
+import type { ExactlyMarketProps } from '../common/exactly.token-fetcher';
+import type { Market } from '../contracts';
+import { EXACTLY_DEFINITION } from '../exactly.definition';
+
+const group = EXACTLY_DEFINITION.groups.deposit;
+const network = Network.ETHEREUM_MAINNET;
+
+@Register.TokenPositionFetcher({ appId: EXACTLY_DEFINITION.id, groupId: group.id, network })
+export class EthereumExactlyDepositTokenFetcher extends ExactlyTokenFetcher {
+  groupLabel = group.label;
```

Comment:
the group label is still a required field, so we cannot remove it from the definition file, and i wouldn't like to have the hardcoded label string duplicated in the codebase. do you have any github issue tracking this deprecation?

Reply:
There's not Github issue indicating the deprecation. We are in the process of making a clear split between the app definitions and the positions. Even if it's not ideal to duplicate strings, I still suggest you do so because if you don't I'll have to do it eventually.